text/0000-pack-inspect-image.md

```diff
@@ -0,0 +1,96 @@
+# Meta
+[meta]: #meta
+- Name: Inspect Image
+- Start Date: 2019-04-25
+- CNB Pull Request: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+An `inspect-image` command will be added to the `pack` CLI, allowing buildpack users to easily view metadata about an image generated by the Cloud Native Buildpack Lifecycle. The command will print the stack ID, stack metadata (run-image and run-image-mirrors), the buildpacks that contributed to the image, the bill of materials, and the processes types.
```

Comment:
This may be out of scope for this RFC, but as we're starting to get more and more of the hyphenated commands, I wonder if we should consider other command patterns, such as:

* `pack [topic]:[command]` (in this case `pack image:inspect`
* `pack [topic] [command]` (in this case `pack image inspect` (similar to `docker`)

I'm not necessarily against hyphenated commands, I just want to make sure we're choosing that pattern for good reasons.

Reply:
I'm open to the idea of changing the pattern of all of our cli commands, but it seems like that should be a different RFC or issue. I wouldn't want the implementation of this RFC to be blocked on that discussion.

Reply:
yea, that's fair